src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/Output/StandardColumnDataGenerator.cs

```diff
@@ -62,6 +62,34 @@ internal class StandardColumnDataGenerator : IStandardColumnDataGenerator
             }
         }
 
+        private string RenderLogMessage(LogEvent logEvent)
+        {
+            var logMessage = logEvent.RenderMessage(_formatProvider);
+            var maxAllowedMessageLength = _columnOptions.Message.DataLength;
+
+            if (maxAllowedMessageLength > 0 && logMessage.Length > maxAllowedMessageLength)
```

Comment:
Lines 70 -76 are the same code as lines 84 - 90. Please consider adding a private method a call it in both places.

Reply:
I use this sort of common function across many projects. Could easily be adapter to include a suffix.

```csharp
    public static class StringExtensions
    {
        /// ...
        public static string Truncate(this string value, int maxLength)
        {
            if (value == null) return null;
            return value.Length <= maxLength ? value : value.Substring(0, maxLength); 
        }
    }
```

Reply:
More or less as provided by the example, I added a `Truncate` extension method. But still opted for an intermediate call to make sure that the default use case (no truncate at all) is handled as quickly as possible without any need to go through the `Truncate` method.